src/main/resources/META-INF/mods.toml

```diff
@@ -58,6 +58,13 @@ Adds Homestuck to your Minecraft!
     # Side this dependency is applied on - BOTH, CLIENT or SERVER
     side="BOTH"
 
+[[dependencies.minestuck]]
+    modId="geckolib3"
+    mandatory=true
+    versionRange="[3.0.32]"
```

Comment:
Suggesting a wider version range, such as `[3.0,)` or `[3.0,4.0)`.
(The first suggestion denotes "any version after and including 3.0", while the second will be the same, but excluding 4.0 and anything past that)